packages/bpk-foundations-common/base/colors/aliases.json

```diff
@@ -77,6 +77,9 @@
     "BLACK_ALPHA_20": "#00000033",
     "BLACK_ALPHA_70": "#000000B3",
     "BLACK_ALPHA_80": "#000000CC",
-    "BABY_BLUE": "#E3F0FF"
+    "BABY_BLUE": "#E3F0FF",
```

Comment:
Checking the Figma design we aren't adding this colour to Backpack

Reply:
Hi @olliecurtis This color existed before. This PR just adds 3 new colors.

Reply:
Oh yeah 🤦I now see that it,s just adding a `,` at the end! My mistake, coffee not consumed yet!